docs/kraken-x3-z3-guide.md

```diff
@@ -28,6 +28,9 @@ In addition to this, Kraken Z coolers restore the embedded fan controller that i
 
 Kraken 2023 AIOs use the same pump and as their Z3 predecessor but the integrated led controller has been removed. The LCD resolution is 240x240 for the standard version and 640x640 for the elite one.
 
+## NZXT Kraken 2024 Elite, Elite RGB
```

Comment:
Are we sure that RGB and non RGB have the same product ID?

Reply:
Would it make more sense to only specify the RGB model at the moment?

Reply:
This line still mentions the standard Elite model, though?